scio-smb/src/main/scala/org/apache/beam/sdk/extensions/smb/ParquetTypeFileOperations.scala

```diff
@@ -46,12 +48,16 @@ object ParquetTypeFileOperations {
   ): ParquetTypeFileOperations[T] =
     ParquetTypeFileOperations(compression, new SerializableConfiguration(conf), null)
 
+  def apply[T: Coder: ParquetType](predicate: FilterPredicate): ParquetTypeFileOperations[T] =
+    apply(predicate, new Configuration())
```

Comment:
I reckon we can define here a `DefaultConfiguration` as well and use it in `ParquetTypeSortedBucketIO`